database/migrations/2020_09_24_131013_slugs-and-geo.php

```diff
@@ -14,10 +14,13 @@ class SlugsAndGeo extends Migration
     public function up()
     {
         Schema::table('entities', function (Blueprint $table) {
-            $table->string('slug')->after('name');
+            $table->string('slug')->after('name')->default('welcome-to-');
             $table->text('geo')->nullable()->after('type');
```

Comment:
In theory observer should be making slug before save gets called, so shouldn't be a case it has to utilise a default 🤔 (Granted, i've yet to test with sql lite)

Reply:
@thybag I am not sure why SQLite decided to complain here, since I do think we have `string` that are not nullable else where. We can detect what database is used and if it's SQLite append `->default()`.

Reply:
Had a play with testing earlier just re: checking validation on the entity/image/maps and ran in to the same issue. Seems there are a few compatibility quirks with schema SQLite - ended up finding a few tables updates didn't go thru when they were mixed with update/column drops etc.

I think mysql only complains on changes to a populated table, where sqllite doesn't take that in to account.